CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+CHANGES IN VERSION 1.22.0 (FROM 1.21.0)
+===================================
+
+Appium 1.22.0 is a minor release
+
+This release is the **finial** release for Appium 1.x seriese.
+We're going to move to Appium 2.x [TODO: link to documentation].
```

Comment:
https://github.com/appium/appium/blob/2.0/docs/en/about-appium/1.x-eol.md

Reply:
not sure about it. I assume we'd still need to do releases if XCTest introduces a breaking update or a major bug is found until final deprecation

Reply:
ah, good point

Reply:
Tweaked the description -> https://github.com/appium/appium/pull/15640/commits/d9c67f726ca1d9514b2dbbd98e064f49562a94de